ingest/rules/nextclade.smk

```diff
@@ -0,0 +1,54 @@
+"""
+This part of the workflow handles running Nextclade on the curated metadata
+and sequences to split the sequences into L and S segments.
+
+REQUIRED INPUTS:
+
+    metadata    = data/subset_metadata.tsv
+    sequences   = "results/sequences_all.fasta"
+
+OUTPUTS:
+
+    metadata        = results/metadata_{segment}.tsv
+    sequences       = results/sequences_{segment}.fasta
+
+See Nextclade docs for more details on usage, inputs, and outputs if you would
+like to customize the rules:
+https://docs.nextstrain.org/projects/nextclade/page/user/nextclade-cli.html
+"""
+
+rule run_nextclade_to_identify_segment:
+    input:
+        metadata = "data/subset_metadata.tsv",
+        sequences = "results/sequences_all.fasta",
+        segment_reference = config["nextclade"]["segment_reference"],
+    output:
+        sequences = "results/sequences_{segment}.fasta",
+    params:
+        min_seed_cover = config["nextclade"]["min_seed_cover"],
+    shell:
+        """
+        nextclade run \
```

Comment:
I'm curious what the decision process was around `nextclade run` versus `augur align`

Reply:
Good question! At least in dengue, `nextclade run` ran much faster than `augur align` from [slide 14 here](https://j23414.github.io/2024-03-12_Nextstrain_Gene_Trees.pdf).

<img width="956" alt="Screenshot 2024-07-26 at 2 14 32 PM" src="https://github.com/user-attachments/assets/4b69b5de-7fd2-4eff-a024-a38d3a4256a0">

I admit I haven't done the same comparison for lassa data, mostly just wanted one that worked and didn't seem noticeably slow.